client/my-sites/plans-features-main/index.jsx

```diff
@@ -501,6 +515,7 @@ export default connect(
 	( state, props ) => {
 		const siteId = get( props.site, [ 'ID' ] );
 		const currentPlan = getSitePlan( state, siteId );
+		const currentPurchase = getCurrentPlan( state, siteId );
```

Comment:
Does this really work? `getCurrentPlan` returns the site's _plan_, but we're treating it as _purchase_, which is a quite different object? I can purchase the same plan once as IAP, once with another method. It will be two different purchases, but one plan.

Reply:
Yep - this function uses data that is populated from `/rest/v1.3/sites/{site_id}/plans. That endpoint grabs data from a bunch of places and (interestingly) lists the subscription ID for the `ID` property 😅  e.g.
```
{
  (plan_id): {
    ID: (subscription_id),
    // other properties
  },
}
```

Reply:
Gotcha, updated! 😄